src/test/java/uk/gov/digital/ho/hocs/search/api/CaseDataServiceTest.java

```diff
@@ -55,10 +64,10 @@ public class CaseDataServiceTest {
 
     @BeforeEach
     public void setup() {
-        caseDataService = new CaseDataService(elasticSearchClient, 10);
+        caseDataService = new CaseDataService(objectMapper, elasticSearchClient, caseTypeMappingRepository);
     }
 
-    @Test
+    /*@Test
```

Comment:
is this in purpose?

Reply:
I knew there was somewhere I missed tests from 😆 fixed...